data-raw/i18n_translations.yml

```
@@ -56,115 +56,143 @@ button:
   runcode:
     en: "Run Code"
     fr: "Lancer le Code"
+    es: "Ejecutar código"
     emo: "🏃"
   runcodetitle:
     # {{kbd}} — The keyboard shortcut that triggers the runcode button
     en: "$t(button.runcode) ({{kbd}})"
     fr: "$t(button.runcode) ({{kbd}})"
+    es: "$t(button.runcode) ({{kbd}})"
     emo: "$t(button.runcode) ({{kbd}})"
   hint:
     en: "Hint"
     fr: "Indication"
+    es: "Pista"
     emo: "💡"
   hint_plural:
     en: "Hints"
     fr: "Indications"
+    es: "Pistas"
     emo: "$t(button.hint)"
   hinttitle:
     en: "$t(button.hint)"
     fr: "$t(button.hint)"
+    es: "$t(button.hint)"
     emo: "$t(button.hint)"
   hintnext:
     en: "Next Hint"
     fr: "Indication Suivante"
+    es: "Siguiente pista"
     emo: ~
   hintprev:
     en: "Previous Hint"
     fr: "Indication Précédente"
+    es: "Pista anterior"
     emo: ~
   solution:
     en: "Solution"
     fr: "Solution"
+    es: "Solución"
     emo: "🎯"
   solutiontitle:
     en: "$t(button.solution)"
     fr: "$t(button.solution)"
+    es: "$t(button.solution)"
     emo: "$t(button.solution)"
   copyclipboard:
     en: "Copy to Clipboard"
     fr: "Copier dans le Presse-papier"
+    es: "Copiar al portapapeles"
     emo: "📋"
   startover:
     en: "Start Over"
     fr: "Recommencer"
+    es: "Reiniciar"
     emo: "⏮"
   startovertitle:
     en: "$t(button.startover)"
     fr: "$t(button.startover)"
+    es: "$t(button.startover)"
     emo: "Start Over"
   continue:
     en: "Continue"
     fr: "Continuer"
+    es: "Continuar"
     emo: "✅"
   submitanswer:
     en: "Submit Answer"
     fr: "Soumettre"
+    es: "Enviar respuesta"
     emo: "🆗"
   submitanswertitle:
     en: "$t(button.submitanswer)"
     fr: "$t(button.submitanswer)"
+    es: "$t(button.submitanswer)"
     emo: "Submit Answer"
   previoustopic:
     en: "Previous Topic"
     fr: "Chapitre Précédent"
+    es: "Tema anterior"
     emo: "⬅"
   nexttopic:
     en: "Next Topic"
     fr: "Chapitre Suivant"
+    es: "Tema siguiente"
     emo: "➡"
   questionsubmit:
     en: "$t(button.submitanswer)"
     fr: "$t(button.submitanswer)"
+    es: "$t(button.submitanswer)"
     emo: "$t(button.submitanswer)"
   questiontryagain:
     en: "Try Again"
     fr: "Réessayer"
+    es: "Volver a intentar"
     emo: "🔁"
 text:
   startover:
     en: "Start Over"
     fr: "Recommencer"
+    es: "Reiniciar"
     emo: "⏮"
   areyousure:
     en: "Are you sure you want to start over? (all exercise progress will be reset)"
     fr: "Êtes-vous certains de vouloir recommencer? (La progression sera remise à zéro)"
+    es: "¿Estás segure que quieres empezar de nuevo? (todo el progreso del ejercicio se perderá)
```

Am I understanding correctly that `segure` is used here as a gender-neutral `seguro/segura`? Would it make sense to rephrase the statement to avoid a gendered adjective? Maybe something like

```suggestion
    es: "¿De verdad quieres empezar de nuevo? (todo el progreso del ejercicio se perderá)
```

---

In Spanish, you have non-sexist language that would be the use _seguro/segura_ but still considers the gender as binary, then you have inclusive language that would be using _segure_, but that is controversial, if you want to avoid controversy, then the option without gender mark you propose is correct.